### ISSUE TITLE
resolve a path

### ISSUE THREAD

**Issue by chase-moskal** [guybedford/es-module-shims#88]
hello! i'm a huge fan of es-module-shims and everything you do @guybedford!

is there a way i can use es-module-shims to resolve an arbitrary path?

i want to load a json file from an import map'd package:

```js
  //                 (can i get es-module-shims to resolve this path?)
  //                                      ↓
const packageFetch = await fetch("metalshop/package.json")

const {version} = await packageFetch.json()
console.log(`metalshop version: ${version}`)
```

👋 thanks

**Comment by chase-moskal** [comment]
i'm imagining something like

```js
const packageFetch = await fetch( importShim.resolve("metalshop/package.json") )
```

**Comment by guybedford** [comment]
@chase-moskal sure this is what `import.meta.resolve` is for. I've put together a PR in https://github.com/guybedford/es-module-shims/pull/89, let me know what you think.

**Comment by chase-moskal** [comment]
holy moly! so fast! @guybedford you are a hero and a champion, thank you! :beers:

**Comment by guybedford** [comment]
Released in 0.5.2. @chase-moskal you're very welcome!
